src/asset.ts

```diff
@@ -0,0 +1,56 @@
+import * as vscode from 'vscode';
+import * as path from 'path';
+import * as fs from 'fs';
+import { Utility } from './utility';
+
+export default class Asset {
+    public constructor(private context: vscode.ExtensionContext) {
+    }
+
+    public getImageUrl(): vscode.Uri | string {
+        let image: vscode.Uri | string = '';
+        if (this.isUseConfigImage()) {
+            image = this.getRandomOne(this.getConfigImages());
+        } else {
+            image = vscode.Uri.file(this.getRandomOne(this.getDefaultImages())).with({ scheme: 'vscode-resource' });
+        }
+        return image;
+    }
+
+    public getTitle(): string {
+        return Utility.getConfiguration().get<string>('showTitle', '小哥哥，小哥哥，代码写久了，该休息啦~');
+    }
+
+    protected getRandomOne(images: string[]): string {
+        var n = Math.floor(Math.random() * images.length + 1) - 1;
+        return images[n];
+    }
+
+    protected getConfigImages(): string[] {
```

Comment:
A 'nice-to-have' feature: support image file in local folder

Reply:
cool idea . I am modifying it now.

Reply:
I tried it, Cannot load resource outside of protocol root,  so Can’t set the path by the custom。